ci/bin/fly_pipeline.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+# Sets up a new deployment pipeline
+# Make sure to set the correct environment variable (sandbox, cicd, staging or prod)
+
+set -euo pipefail
+
+: ${WORKSPACE:="bpm"}
+: ${ENVIRONMENT:="sandbox"}
+: ${CONFIGURATION:="mail"}
```

Comment:
Probably don't need to do this as a var - it's only really relevant for the statement of record where you can deploy with multiple different configurations. For this you can probably just hardcode an identifier (so in the `pipeline` bit below it would be:
```
pipeline="${ENVIRONMENT}-${WORKSPACE}-deploy-mail-lambda"
```
)

Reply:
I think I'll leave it as a var, if you don't mind